components/eam/src/physics/cam/gw_drag.F90

```diff
@@ -118,6 +118,11 @@ module gw_drag
   ! namelist 
   logical          :: history_amwg                   ! output the variables used by the AMWG diag package
 
+  logical  :: use_gw_convect_old          ! switch to enable legacy behavior
+  real(r8) :: gw_convect_plev_src_wind    ! reference pressure level for source wind for convective GWD
```

Comment:
Add that gw_convect_plev_src_wind must be in units Pa.

Reply:
fixed

Reply:
@whannah1 , do we have simulations that documents  the behavior  with the new Beres scheme for heating depth etc. (i.e.,  use_gw_convect_old = .false.).